lib/graphql/payment-processor-adapters/heroku.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+const herokuAdapter = {
+  register: function(signature, email) {
+
+    // NB I think this is what we need from BitCore
```

Comment:
If public key is needed and ECDSA, it will be better to use https://www.npmjs.com/package/secp256k1 since the crypto is faster, as we're currently using to verify signatures and etc. Should crypto methods should be implemented here in service-storage-models itself, or elsewhere?